src/YourEasyRent/DataBase/ProductRepository.cs

```diff
@@ -101,5 +102,21 @@ public async Task UpsertManyProducts(IEnumerable<Product> products)
                 await UpsertProduct(product);  
             }
         }
+
+        public async Task<List<string>> GetBrandForMenu(int limit)
+        {
+            var pipeline = new List<BsonDocument>() 
```

Comment:
Good job. 
You can also use a simpler query.

Something like this:

 var distinctBrands = await _productCollection.DistinctAsync<string>("Brand", FilterDefinition<BsonDocument>.Empty);
 
 You just get all unic brand fields from db. I'm not sure about details you might need to test it in compass.

Reply:
did it

Reply:
did it